src/convertToObject.js

```diff
@@ -6,7 +6,13 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const strings = sourceString
+    .split(';')
+    .filter((string) => string.includes(':'))
+    .map((string) => string.split(':'))
+    .map(([name, value]) => ({ [name.trim()]: value.trim() }));
```

Comment:
[CODE STYLE] - All chained methods called on the same level should have same indentation. This improves code readability. Please adjust the indentation of the chained methods to align with the first method in the chain.

GOOD EXAMPLE:
const cssStyles = styles
 .map(style => style)
 .filter(style => style.length)
 .slice(0, 5)